addons/sale_analytic_plans/sale_analytic_plans.py

```diff
@@ -38,4 +40,17 @@ def invoice_line_create(self, cr, uid, ids, context=None):
         return create_ids
 
 
+class sale_order(osv.Model):
+    _inherit = 'sale.order'
+    
+    def fields_view_get(self, cr, user, view_id=None, view_type='form', context=None, toolbar=False, submenu=False):
+        res = super(sale_order, self).fields_view_get(cr, user, view_id, view_type, context, toolbar=toolbar, submenu=submenu)
```

Comment:
if there is a missing dependency, you add a link module, you do not hack fields_view_get